crates/aptos/src/list/mod.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! A command to list resources owned by an address
+//!
+//! TODO: Examples
+//!
+
+use aptos_types::account_address::AccountAddress;
+use clap::{Parser};
+use crate::CliResult;
+use reqwest;
+
+/// Command to list resources owned by an address
+///
+#[derive(Debug, Parser)]
+pub struct ListResources {
+    /// Address of account you want to list resources for
+    account: AccountAddress,
+    #[clap(long, default_value = "https://fullnode.devnet.aptoslabs.com")]
+    node_url: String,
+}
+
+impl ListResources {
+    async fn get_resources(self) -> Result<Vec<serde_json::Value>, reqwest::Error> {
+        reqwest::get(format!("{}/accounts/{}/resources", self.node_url, self.account)).await?.json().await
+    }
+
+    // TODO: Format this in a reasonable way while providing all information
+    // add options like --tokens --nfts etc
+    fn json_string(vec: Vec<serde_json::Value>) -> String {
+        let mut output: String = "".to_owned();
+        for json in &vec {
+            output.push_str(&format!("{}\n", json.to_string()));
+        }
+        output.to_string()
+    }
+    
+    pub async fn execute(self) -> CliResult {
+        match self.get_resources().await {
+            Ok(result) => Ok(ListResources::json_string(result)),
+            Err(error) => Err(error.to_string())
+        }
```

Comment:
You can also just simply use the convenience function I have elsewhere, as well as map to compile the string out of it.  It's serialized to a JSON string, so it should just show up as a list (unless there's some weird formatting)

```
to_common_result(self.get_resources())
```

This will make it something like: 
```
{
    "Result": [
        "item1":"...",
        "item2":"..."
    ]
}
```

Reply:
I still had to match because `to_common_result` and `CliResult` requires `Result<_, Err: Common:Types::Error>` and the error from the reqwest is reqwest::Error.

But maybe there is a fancier way to do this.

Reply:
```
.map_err(|err| Error::UnexpectedError(err.to_string()))
```

Reply:
beautiful